Project/Source/Components/UI/ComponentCanvasRenderer.cpp

```diff
@@ -22,12 +22,12 @@ void ComponentCanvasRenderer::Render(const GameObject* gameObject) const {
 
 		//IF OTHER COMPONENTS THAT RENDER IN UI ARE IMPLEMENTED, THEY MUST HAVE THEIR DRAW METHODS CALLED HERE
 		ComponentImage* componentImage = gameObject->GetComponent<ComponentImage>();
-		if (componentImage != nullptr) {
+		if (componentImage != nullptr && componentImage->IsActiveInHierarchy()) {
```

Comment:
Change IsActiveInHierarchy to IsActive (so it only checks that the Component is Active). In ModuleUserInterface->Render already checks that the Owner of the ComponentCanvasRenderer is ActiveInHierarchy

Reply:
Thanks!